tests/queries/0_stateless/00515_enhanced_time_zones.sql

```diff
@@ -68,4 +68,4 @@ DETACH TABLE tab;
 ATTACH TABLE tab SETTINGS allow_nonconst_timezone_arguments = 0; -- { serverError ILLEGAL_COLUMN }
 ATTACH TABLE tab SETTINGS allow_nonconst_timezone_arguments = 1;
 
-DROP TABLE tab;
+DROP TABLE tab SYNC;
```

Comment:
In Stress tests, a queries file may be executed partially, `SYNC` will not help if the `DROP TABLE` query is not executed at all